File: nos2jdbc-tutorial-spring/src/main/java/nos2jdbc/tutorial/spring/entity/nonauto/MemberKey.java
package nos2jdbc.tutorial.spring.entity.nonauto;

import java.util.List;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.OneToMany;
import nos2jdbc.annotation.NonAuto;

@Entity @NonAuto
public class MemberKey {
    @Id
    public Long memberId;
    
    public String memberName;
    
    @OneToMany(mappedBy = "member")
    public List<MemberYm> ymList;
    

}

File: nos2jdbc-tutorial-kotlin-spring/src/main/kotlin/nos2jdbc/tutorial/kotlinspring/entity/nonauto/MemberKey.kt
package nos2jdbc.tutorial.kotlinspring.entity.nonauto

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.OneToMany

import nos2jdbc.annotation.NonAuto

@Entity @NonAuto
class MemberKey {
    @Id
    var memberId: Long? = null
    
    var memberName: String? = null
    
    @OneToMany(mappedBy = "member")
    var ymList: List<MemberYm> = mutableListOf()
    

}
